test/types/string/kbrady/search.good

```diff
@@ -5,116 +5,116 @@ d: çççççççççççççççççççççççççççç
 
 x.___("Hello")
 count: 1
-find: 1
-rfind: 1
+find: (_bindex = 1)
```

Comment:
Is this formatting just the default (compiler-provided) print format for a ByteIndex?  I think we should prettify this (accomplished by implementing the writeThis() or readWriteThis() methods on the record), perhaps just making it print like a raw integer as before?

Reply:
Yes, this is just the default.  I'm OK with making them look like integers.  The one disadvantage to that is that I found it nice for debugging to see explicitly that it was a byteIndex.

Reply:
Maybe we could have a config param that used a more verbose output when debugging was desired?  I think for the common case, an integer seems nicer.

Or if we did want to distinguish them in some way, I'd come up with some sort of format that looks a bit more user-friendly than the compiler's default... but that strikes me as a challenging design problem (where my head goes to verbose options like `byteIndex(1)` or `1:byteIndex`...  I'm just not sure this is what I'd want to see by default / all the time).

Reply:
I said `config param` but `config const` might be more appropriate given that I/O isn't particularly performance-critical and it would save recompiling.

Reply:
After thinking about it, the distinction was really more useful to me than it would be to anyone else.  I'd be OK with just printing them as integers and leaving it at that.